app/bundles/CoreBundle/Command/CleanupMaintenanceCommand.php

```diff
@@ -25,6 +25,7 @@ public function __construct(
         PathsHelper $pathsHelper,
         CoreParametersHelper $coreParametersHelper
     ) {
+    	$this->coreParametersHelper = $coreParametersHelper;
```

Comment:
Just add `private` keyword to the line 26 and remove this line to avoid this static analysis error.

Reply:
Done.